### ISSUE TITLE
User Review Checklist

### ISSUE THREAD

**Issue by gkwan-ibm** [OpenLiberty/draft-guide-cloud-openshift-operator#38]
User Review:

Allow a person with a fresh mind to read the guide and execute the steps described in the guide, test whether each step is clear enough to understand, and no logic errors.

Review and walkthrough by a non-SME or a new user with a critical and fresh pair of eyes:

- [ ] Record the time it takes to go through the entire guide
- [ ] Test each step in the guide is clear and easy to understand
- [ ] Ensure no logic errors
- [ ] Report anything that does not work
- [ ] Report anything usability issues (confusing or hard to follow)
- [ ] Does the guide achieve basic understanding/working knowledge of the topic in the guide?

**Comment by InfTkm** [comment]
Time: 30 - 60 minutes depending on how well the user is experienced with OpenShift.  

1. 
<img width="1781" alt="Screen Shot 2020-09-24 at 1 46 36 PM" src="https://user-images.githubusercontent.com/25573786/94206226-f1955f00-fe92-11ea-8fa9-965df19bdba8.png">
I wish I can separately adjust the width of each column. Or don't let the rightmost column take too much space because usually I just click the copy button don't read it so carefully. My attention is mostly on the middle column. <br/>
2. It would be better if there is a hyperlink provided to explain Strimzi Operator when it is mentioned the first time. There is a link in the document, but it's placed nearby the second occurrence of the term.     <br/>
3. git clone link is still in draft (I assume this will be changed once in production)  <br/>
4. 
<img width="743" alt="Screen Shot 2020-09-24 at 6 29 45 PM" src="https://user-images.githubusercontent.com/25573786/94206645-f27ac080-fe93-11ea-89c4-0f8ad82efaa3.png">
This is a great design. For some reason, I failed to realize the file it refers to is in the right-hand-side column.
I guess there are two reasons. First is that the word here says 'Create' instead of 'Copy', so I assumed that I should create a new one myself.  
Second reason is that there is a hyperlink below saying the configuration is based on that file, so I copied the file from the hyperlink, which is incorrect.
But as soon as I get the design, I will never miss it again.

Overall, a solid user guide, just a few minor things.

**Comment by AustinSeto** [comment]
1. That's a feature of the OpenLiberty.io website. The table of contents on the left should have an orange X button to let you close/minimize it which will readjust the columns giving you more space to view the guide content in the other columns. 

2. Moved link to Strimzi Operator website to introduction when Strimzi is first described

3. Yes, the guide repository name will be updated upon publishing so the clone command described will work

4. Made a minor wording update to clarify the hyperlink

5. Added clarification on Operator installation

***

Changes are in #40
